cmd/neofs-node/config.go

```diff
@@ -837,7 +837,21 @@ func (c *cfg) handleLocalNodeInfo(ni *netmap.NodeInfo) {
 // bootstrap sets local node's netmap status to "online".
 func (c *cfg) bootstrap() error {
 	ni := c.cfgNodeInfo.localInfo
-	ni.SetOnline()
+
+	// switch to online except when under maintenance
+	if st := c.cfgNetmap.state.controlNetmapStatus(); st == control.NetmapStatus_MAINTENANCE {
+		ni.SetMaintenance()
+
+		c.log.Info("bootstrap with untouched node state",
+			zap.Stringer("current", st),
+		)
+	} else {
+		ni.SetOnline()
+
+		c.log.Warn("change node state to bootstrap with",
```

Comment:
Will we have this message if the node is already online?

Reply:
Changed to info level.